### DISCUSSION TITLE
How to fix `spelling error` warnings that displayed when using localized version of Visual Studio.

### DISCUSSION THREAD

**Discussion by filzrev** [dotnet/docfx#9544] [Show and tell]
When using localized version of Visual Studio.
A lot of `SPELL` category intellisense warnings are displayed when build `docfx` project.

These problems can be resolved by following steps.
1. Install Visual Studio `English` language pack.
2. Run `Install-Language en-us` command on Visual Studio developer PowerShell prompt. (With elevated privileges)

Without `Step 2.` spell check feature is not works as expected.

**Comment by NattyXO** [comment]
1. **Install Visual Studio English Language Pack:**
   Go to the Visual Studio Installer and look for the English Language Pack. Install it alongside your localized version of Visual Studio. This pack provides English language support, including spell checking functionality.

2. **Run Visual Studio Developer PowerShell Prompt as Administrator:**
   Open the Visual Studio Developer PowerShell Prompt with elevated privileges (Run as Administrator).

3. **Run Install-Language Command:**
   In the PowerShell prompt, execute the following command:
   ```
   Install-Language en-us
   ```
   This command installs the English language components needed for proper spell checking in Visual Studio.

4. **Restart Visual Studio:**
   Close and reopen Visual Studio for the changes to take effect.

After completing these steps, the English language components should be installed, enabling proper spell check functionality even in a localized version of Visual Studio. This should mitigate the SPELL category IntelliSense warnings that were occurring during the build of your DocFX project.

Remember, the key is to ensure that the English language support is installed and configured properly alongside your localized version of Visual Studio to enable accurate spell checking and resolve the associated warnings.
